### PR TITLE
Failover Database Readable Secondary

### PR DESCRIPTION
**Pull request by emgu-ms**
### Latest improvements:
Add swagger changes for failover databases readable secondary
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by jaredmoo — APPROVED** [review]

**Comment by emgu-ms** [comment]
Ready to be merged as its now been deployed to some clusters

File: specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json [hunk 1]
```diff
@@ -37,6 +37,21 @@
             "required": true,
             "type": "string"
           },
+          {
```
Comment (sergey-shandar) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json]: Please, append new parameters to the end of the parameter array.

Comment (emgu-ms) [reply specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/FailoverDatabases.json]: This is a generated file so we can't move this to the end of the parameter array

**Review by KrisBash — APPROVED** [review]
Signing off

**Pull request merged**